### ISSUE TITLE
ColladaLoader not importing animations

### ISSUE THREAD

**Issue by aymen157** [mrdoob/three.js#18035]
##### Description of the problem

[riven.zip](https://github.com/mrdoob/three.js/files/3907078/riven.zip)
The mesh loads and everything, and the animation's tracks doesn't and its duration is 0, this happens on some models

##### Three.js version

- [ ] Dev
- [x] r111
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by Mugen87** [comment]
Um, I'm not able to unzip your file on my iMac. Can you also share the file via e.g. Google Drive?

**Comment by IMHOJEONG** [comment]
https://drive.google.com/file/d/1MyqPLgMsxAQ5THsg-uRkL1pchovFBa_e/view?usp=sharing

**Comment by Mugen87** [comment]
The loader can't handle the file because it is unable to parse nested animations: #13981

Besides, the loader only supports the `matrix` transform type. Your file however does use different types like `scale`. 

I think it's unlikely that the loader will support these features in the time so it's probably better to convert your asset to `glTF` instead.

https://github.com/KhronosGroup/COLLADA2GLTF

Or, if possible, export to `glTF` in the first place.

**Comment by aymen157** [comment]
Yep, i did, but is there a planned support for this 'type' of files?

**Comment by Mugen87** [comment]
Um, probably because of the special transform types. I've once tried to convert the old `pump` Collada model which was used in our [keyframe animation example](https://threejs.org/examples/webgl_animation_keyframes). We had to completely replace it because I've found no tool that could convert the model to `glTF`. 

Unfortunately, this part of the Collada spec is very complex and hard to map on `three.js`'s animation system. Hence, I doubt `ColladaLoader` will ever be able to handle such files.
